src/calendar-list/index.tsx

```diff
@@ -192,7 +195,7 @@ class CalendarList extends Component<CalendarListProps, CalendarListState> {
   }
 
   scrollToDay(d: XDate, offset: number, animated: boolean) {
-    const {horizontal, calendarHeight, calendarWidth, pastScrollRange, firstDay} = this.props;
+    const {horizontal, calendarHeight = 360, calendarWidth = width, pastScrollRange = 50, firstDay} = this.props;
```

Comment:
same here, let's declare const values at the top file for better readability..

Reply:
Done